app/assets/javascripts/invoice.js

```diff
@@ -56,8 +56,9 @@ $(document).on('change',".selectclient", function() {
 
         },
         success: function (data) {
-            var str = data.street_1+" "+data.street_2+", <br/>"+data.city+", "+data.state+", <br/>"+data.pincode.to_s
-            $("#client_address").html(str);
+            var stateString = data.state
+            var addresssString = data.street_1+" "+data.street_2+", <br/>"+data.city+", "+stateString+", <br/>"+data.pincode.toString();
```

Comment:
`stateString` is used only once withing the callback function. Avoid creating local variables.